plugin_interpreter/src/rethink_interface.py

```diff
@@ -192,22 +177,23 @@ def _log_db_error(self, err, logger):
     def _database_init(self, logger):
```

Comment:
You can take out the logger argument and replace the `logger` with `self.logger` wherever applicable in this function.